streams/src/main/java/org/apache/kafka/streams/state/internals/Segment.java

```diff
@@ -39,6 +41,13 @@ public int compareTo(final Segment segment) {
         return Long.compare(id, segment.id);
     }
 
+    @Override
+    protected TableFormatConfig getTableConfig() {
```

Comment:
For windowed stores, don't enable bloom filters

Reply:
>Why? It is about range queries? Re-call that we convert range-queries into multiple point-lookups.

I took a cursory look at the code, but you raise a good point.  Overall I'm thinking maybe we need to see if Bloom-filters affect range queries and if not, maybe just enable them across the board.